<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSubscriptionWindow</class>
 <widget class="QMainWindow" name="CreateSubscriptionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>611</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Створення абонемента&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createSubscriptionPushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Створити абонемент</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>611</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="days">
       <property name="text">
        <string>Дні відвідувань</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Графік роботи (кінець)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Графік роботи (початок)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_1">
         <property name="text">
          <string>Понеділок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Вівторок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>Середа</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string>Четвер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_5">
         <property name="text">
          <string>П'ятниця</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_6">
         <property name="text">
          <string>Субота</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_7">
         <property name="text">
          <string>Неділя</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="priceLineEdit"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="trainingAmountLineEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="price">
       <property name="text">
        <string>Ціна</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="title">
       <property name="text">
        <string>Назва</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="3" column="2">
      <widget class="QTimeEdit" name="timeEdit">
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="trainingAmount">
       <property name="text">
        <string>Кількість тренувань</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QTimeEdit" name="timeEdit_2">
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="unlimitedRadioButton">
         <property name="text">
          <string>Безлімітний</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="limitedRadioButton">
         <property name="text">
          <string>Обмежений</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
